variables.tf

```diff
@@ -0,0 +1,142 @@
+# Basic Variables
+variable "profile" {
+  default = "the AWS profile to use"
+}
+
+variable "vpc_id" {
+  description = "ID of the VPC the security_group should be connected to"
+}
+
+variable "description" {
+  description = "The description of the security group"
+  default = "Basic http Security-Group"
+}
+
+variable "name" {
+  description = "The name of the security group"
+}
+
+variable "custom_tags" {
+  type = "map"
+  default = {}
+}
+
+# Variables for IPv4 cidr_based rules
+variable "cidr_ingress_rules" {
```

Comment:
I think we should 'prefix' this variables with 'ipv4' as well

Reply:
i changed it according to the `ipv6`-ones